src/tracking.rs

```diff
@@ -19,21 +18,21 @@ use crate::types::{GossipMessage, GossipPeerManager};
 pub(crate) async fn download_gossip(persistence_sender: mpsc::Sender<GossipMessage>,
 		completion_sender: mpsc::Sender<()>,
 		network_graph: Arc<NetworkGraph<TestLogger>>) {
-	let mut key = [0; 32];
-	let mut random_data = [0; 32];
-	thread_rng().fill_bytes(&mut key);
-	thread_rng().fill_bytes(&mut random_data);
+	let key = [42; 32];
```

Comment:
are you sure you wanna fixate this? Wouldn't this be running the risk of nodes figuring out that they're connected to an RGS instance?